zfs/versions.go

```diff
@@ -211,6 +211,13 @@ func (o *ListFilesystemVersionsOptions) typesFlagArgs() string {
 }
 
 func (o *ListFilesystemVersionsOptions) matches(v FilesystemVersion) bool {
+	// Workaround when using JobSnapPrefix because bookmarks are not following
+	// the snapshotting prefix but they have fixed prefix:
+	// replicationCursorBookmarkNamePrefix = "zrepl_CURSOR"
+	if len(o.ShortnamePrefix) > 0 && len(o.Types) == 0 && v.Type == Bookmark && strings.HasPrefix(v.Name, "zrepl_CURSOR") {
```

Comment:
We should just allow all bookmarks, probably rename `ShortnamePrefix` to `SnapPrefix`.

Reply:
done, thanks